src/components/patch/PatchContainer.tsx

```diff
@@ -83,45 +85,62 @@ export class PatchContainer extends React.Component<Props, State> {
-          <FormControl className="advanced-select" key="project">
-            <InputLabel>Project</InputLabel>
-            <Select
-              multiple={true}
-              value={this.state.selectedProjects}
-              onChange={this.onProjectSelectChange}
-              renderValue={this.renderSelection}
-              input={<Input className="advanced-input" />}
-              MenuProps={MenuProps}
-            >
-              {this.state.allProjects.map(project => (
-                <MenuItem key={project} value={project}>
-                  <Checkbox checked={this.state.selectedProjects.indexOf(project) > -1} />
-                  <ListItemText primary={project} />
-                </MenuItem>
-              ))}
-            </Select>
-          </FormControl>
-        </div>
+        <Grid container={true} className="filter-container">
+          <Grid item={true} xs={10}>
+            <div className="dropdown-container">
+              <FormLabel className="filter-label">Filter Patches</FormLabel>
+              <FormControl className="advanced-select" key="status">
+                <InputLabel>Status</InputLabel>
+                <Select
+                  multiple={true}
+                  value={this.state.selectedStatuses as string[]}
+                  onChange={this.onStatusSelectChange}
+                  renderValue={this.renderSelection}
+                  input={<Input className="advanced-input" />}
+                  MenuProps={MenuProps}
+                >
+                  {this.state.allStatuses.map(status => (
+                    <MenuItem key={status} value={status}>
+                      <Checkbox checked={this.state.selectedStatuses.indexOf(status) > -1} />
+                      <ListItemText primary={status} />
+                    </MenuItem>
+                  ))}
+                </Select>
+              </FormControl>
+              <FormControl className="advanced-select" key="project">
+                <InputLabel>Project</InputLabel>
+                <Select
+                  multiple={true}
+                  value={this.state.selectedProjects}
+                  onChange={this.onProjectSelectChange}
+                  renderValue={this.renderSelection}
+                  input={<Input className="advanced-input" />}
+                  MenuProps={MenuProps}
+                >
+                  {this.state.allProjects.map(project => (
+                    <MenuItem key={project} value={project}>
+                      <Checkbox checked={this.state.selectedProjects.indexOf(project) > -1} />
+                      <ListItemText primary={project} />
+                    </MenuItem>
+                  ))}
+                </Select>
+              </FormControl>
+            </div>
+          </Grid>
+          <Grid item={true} xs={2} className="expand-container">
+            <div className="expand">
+              <FormControlLabel
+              control={
+                <Checkbox
```

Comment:
A checkbox seems like the wrong UI element for this. It implies that there are only 2 valid states, but actually if I check it and then collapse something, it leaves the UI ambiguous

Reply:
Okay, I've replaced these with expand/collapse buttons. Let me know how they look.